src/lib/draw/route/RouteControls.svelte

```diff
@@ -6,7 +6,7 @@
 
   // TODO Save this in the route
   let avoidDoublingBack = false;
-  $: routeTool.setConfig({
+  $: routeTool.setRouteConfig({
```

Comment:
Sliiight bug here. If we make a snapped area, then make a snapped route, avoid doubling back will be true from the last call, but the checkbox won't be ticked. I think we need to add a property to the output geojson feature to indicate this option and keep it synced.